### ISSUE TITLE
ImportError: cannot import name 'ABCIndexClass' from 'pandas.core.dtypes.generic'

### ISSUE THREAD

**Issue by arademaker** [CODAIT/text-extensions-for-pandas#219]
```
(venv) ar@tenis ~ % python
Python 3.9.5 (default, May  4 2021, 03:36:27)
[Clang 12.0.0 (clang-1200.0.32.29)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import pandas
>>> import text_extensions_for_pandas as tp
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/ar/venv/lib/python3.9/site-packages/text_extensions_for_pandas/__init__.py", line 27, in <module>
    from text_extensions_for_pandas.array.span import (
  File "/Users/ar/venv/lib/python3.9/site-packages/text_extensions_for_pandas/array/span.py", line 33, in <module>
    from pandas.core.dtypes.generic import ABCDataFrame, ABCIndexClass, ABCSeries
ImportError: cannot import name 'ABCIndexClass' from 'pandas.core.dtypes.generic' (/Users/ar/venv/lib/python3.9/site-packages/pandas/core/dtypes/generic.py)
>>>
```

**Comment by cullinap** [comment]
Looks like this relates back to #218

**Comment by frreiss** [comment]
This should be fixed with #218. We will cut a new release in a few days.

**Comment by frreiss** [comment]
In the meantime, you can work around this issue by downgrading Pandas to 1.2.x.
